src/test-helper.js

```diff
@@ -144,7 +145,8 @@ export function createTestDocument(container, user, document) {
       sections: document.sections ?? [],
       tags: document.tags || [],
       appendTo: null,
-      archived: document.archived
+      archived: document.archived,
+      origin: DOCUMENT_ORIGIN.internal
```

Comment:
Could you also for these 2 properties first check if it is defined on `document` and then fall back to the default, we want to be able to override these values.

Reply:
Ah yes, sure, done it for all 3 properties actually. Thanks!